sponsorship_tracking/view/contract_view.xml

```diff
@@ -46,7 +48,7 @@
         </record>
 
         <record id="view_tracking_contract_form" model="ir.ui.view">
-			<field name="name">follow.contract.compassion.form</field>
+			<field name="name">recurring.contract.compassion.form</field>

```

Comment:
Not sure if this is a good renaming. It should not be a name already defined in a parent module (sponsorship_compassion).